diff --git a/node_modules/@strapi/admin/dist/_chunks/index-3L2hpEDi.mjs b/node_modules/@strapi/admin/dist/_chunks/index-3L2hpEDi.mjs
index 9b5b230..83c366c 100644
--- a/node_modules/@strapi/admin/dist/_chunks/index-3L2hpEDi.mjs
+++ b/node_modules/@strapi/admin/dist/_chunks/index-3L2hpEDi.mjs
@@ -13537,7 +13537,7 @@ const DynamicComponent = ({
   dynamicComponentsByCategory = {},
   onAddComponent
 }) => {
-  const [isOpen, setIsOpen] = React.useState(true);
+  const [isOpen, setIsOpen] = React.useState(false);
   const { formatMessage } = useIntl();
   const { getComponentLayout } = useContentTypeLayout();
   const { modifiedData } = useCMEditViewDataManager();
